src/Symfony/Component/Validator/Constraints/UrlValidator.php

```diff
@@ -61,7 +61,13 @@ public function validate($value, Constraint $constraint)
             return;
         }
 
-        $pattern = sprintf(static::PATTERN, implode('|', $constraint->protocols));
+        $protocolPattern = '(%s):';
```

Comment:
non-capturing, i.e. `(?:%s):`? Might be patched in 2.7 for groups currently in `PATTERN`. Not a real blocker though.